__tests__/app.test.js

```diff
@@ -113,3 +113,40 @@ describe("GET api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("should respond with an array of comments with supplised review_id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toHaveLength(3);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            author: expect.any(String),
+            review_id: expect.any(Number),
+            created_at: expect.any(Date),
+          });
+        });
+      });
+  });
+  test("400: Bad request when supplied id is invalid", () => {
+    return request(app)
+      .get("/api/reviews/invalid_id/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("404: Not found when supplied id is valid but does not exist", () => {
+    return request(app)
+      .get("/api/reviews/1000")
```

Comment:
This test does not look quite right - the path should be /api/reviews/1000/comments. You will need to change this and re-run your tests. In order to pass this test, you will need some logic that checks whether this article exists.